FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/FacebookChatTests.cs

```diff
@@ -132,25 +132,25 @@ private void GetEnvironmentVars()
                 _appSecret = Environment.GetEnvironmentVariable("FacebookTestBotFaceBookAppSecret");
                 if (string.IsNullOrWhiteSpace(_appSecret))
                 {
-                    Assert.Inconclusive("Environment variable 'FacebookTestBotFaceBookAppSecret' not found.");
+                    Assert.Fail("Environment variable 'FacebookTestBotFaceBookAppSecret' not found.");
```

Comment:
For all the `inconclusive` to `fail` changes, is there a situation where some of these env vars could not be set and that's the expected behavior? This change could possibly create a false negative.

Reply:
No.